templates/components/common/navigation.html

```diff
@@ -1,7 +1,7 @@
 <nav class="navUser">
-    {{#if customer.id}}
+    {{#or customer (unless theme_settings.restrict_to_login)}}
```

Comment:
Should we really change `customer.id` to just `customer` in this check?

Reply:
@yurytut1993 to maintain the consistency of the project, I decided to change customer.id to customer.